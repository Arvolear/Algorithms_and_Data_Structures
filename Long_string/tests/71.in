9305151323612851252912655737326476081115
969740994826771608144712715