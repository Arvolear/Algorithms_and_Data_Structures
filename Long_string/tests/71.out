9305151323613820993907482508934620793830
9305151323611881511917828965718331368400
9023586701573976969026963281256050198617663048475828178000711877225
9595501657919